maquetado/scss/login.scss

```diff
@@ -0,0 +1,95 @@
+@import "colors";
+
+body {
+  display: flex;
+  justify-content: center;
+}
+
+.login-container {
+  background-color: $light-grey;
+  border-top: 8px solid $light-blue;
+  display: flex;
+  height: 400px;
+  justify-content: center;
+  margin-top: 70px;
+  width: 300px;
+}
+
+.login-inputs-container {
+  display: flex;
+  flex-direction: column;
+  &::before {
+    align-self: center;
+    background: url("../maquetado/image/logoWolox.png") no-repeat center/contain;
+    content: "";
+    height: 70px;
+    margin-top: 20px;
+    width: 200px;
+  }
```

Comment:
This is not a decorative image, so here you should use an img tag instead. After and before are only used for those images that doesn't not have any meaning by themselves. Here we are talking about Wolox's logo